website/templates/project/modal_remove_contributor.mako

```diff
@@ -0,0 +1,122 @@
+<link rel="stylesheet" href='/static/css/pages/remove-contributor-page.css'>
+<div id="removeContributor" class="modal fade">
+    <div data-bind="css: modalSize">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+                <h3 class="modal-title" data-bind="text:pageTitle"></h3>
+            </div>
+            <div class="modal-body" >
+                <div data-bind="if: canRemoveNode() && !pageChanged()">
```

Comment:
Hmmm.  I have seen both used, and am not sure if we have an institutional preference of one over the other.  I feel like I usually see bindings in containers in general on the OSF a bit more then the containerless comment bindings.  @sloria, is there a reason to use one over the other?